measures/ServerDirectoryCleanup/measure.rb

```diff
@@ -11,71 +11,221 @@ def name
   # define the arguments that the user will input
   def arguments(model)
     args = OpenStudio::Ruleset::OSArgumentVector.new
+
+    arg = OpenStudio::Ruleset::OSArgument.makeBoolArgument('in_osm', true)
+    arg.setDisplayName('Retain in.osm')
```

Comment:
I suggest prefixing all of the argument names with `retain_`, otherwise it's not obvious if true means retain or delete.